website/content/docs/k8s/operations/uninstall.mdx

```diff
@@ -29,12 +28,11 @@ Refer to the [Consul K8s CLI reference](/docs/k8s/k8s-cli#uninstall) topic for d
 
 Run the `helm uninstall` **and** manually remove resources that Helm does not delete.
 
+1. (Optional) If Consul is installed in a dedicated namespace, set the kubeConfig context to the `consul` namespace. Otherwise, subsequent commands will need to include `-n consul`.
```

Comment:
```suggestion
1. (Optional) If Consul is installed in a dedicated namespace, set the kubeConfig context to the `consul` namespace. Otherwise, subsequent commands will need to include `--namespace consul`.
```

I could not find the edit on this line, but I guess this should match the example below?